recipe/meta.yaml

```diff
@@ -25,25 +24,27 @@ requirements:
   host:
     - python
     - pip
-    - numpy >=1.13.3,<1.17
+    - numpy >=1.13.3
     - cython
 
   run:
     - python
     - scipy >=1.0.0
-    - numpy >1.13.3,<1.17
-    - matplotlib-base
+    - numpy >1.13.3
+    - matplotlib-base >=1.5.1,<3.4
     - seaborn
```

Comment:
should scikit-learn, seaborn and netcdf4 be there? Aren't they optional dependencies?